seedlet-blocks/functions.php

```diff
@@ -25,7 +25,7 @@ function seedlet_blocks_setup() {
 		add_editor_style( 'style-editor.css' );
 	}
 endif;
-add_action( 'after_setup_theme', 'seedlet_blocks_setup' );
+add_action( 'after_setup_theme', 'seedlet_blocks_setup', 999 );
```

Comment:
Could we change the priority to something that appears more intentional, like 1 after the default?